### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (#mod-login-username invalid)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#53]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[kh - Administration](http://localhost/kh/administrator/)
#### Element path
#mod-login-username
#### Snippet
`<input name="username" id="mod-login-username" type="text" class="form-control form-control-danger invalid" required="required" autofocus="" autocomplete="username" aria-invalid="true">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.96 (foreground color: #de350b, background color: #fdece8, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
